src/main/java/com/hivemq/cli/commands/swarm/run/SwarmRunStartCommand.java

```diff
@@ -0,0 +1,251 @@
+/**
+ * Copyright 2019 HiveMQ and the HiveMQ Community
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.hivemq.cli.commands.swarm.run;
+
+import com.google.common.annotations.VisibleForTesting;
+import com.google.common.io.Files;
+import com.hivemq.cli.MqttCLIMain;
+import com.hivemq.cli.commands.swarm.AbstractSwarmCommand;
+import com.hivemq.cli.commands.swarm.error.Error;
+import com.hivemq.cli.commands.swarm.error.SwarmApiErrorTransformer;
+import com.hivemq.cli.openapi.ApiException;
+import com.hivemq.cli.openapi.swarm.*;
+import okhttp3.HttpUrl;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+import org.tinylog.Logger;
+import picocli.CommandLine;
+
+import javax.inject.Inject;
+import javax.inject.Provider;
+import java.io.File;
+import java.io.IOException;
+import java.io.PrintStream;
+import java.util.Base64;
+import java.util.concurrent.TimeUnit;
+
+/**
+ * @author Yannick Weber
+ */
+@CommandLine.Command(name = "start",
+        description = "Start HiveMQ Swarm runs.",
+        synopsisHeading = "%n@|bold Usage:|@  ",
+        descriptionHeading = "%n",
+        optionListHeading = "%n@|bold Options:|@%n",
+        commandListHeading = "%n@|bold Commands:|@%n",
+        mixinStandardHelpOptions = true,
+        versionProvider = MqttCLIMain.CLIVersionProvider.class)
+public class SwarmRunStartCommand extends AbstractSwarmCommand {
+
+    @CommandLine.Option(names = {"-f", "--file"}, description = "The scenario file. " +
+            "If a scenario file is given this command uploads, executes and deletes the scenario afterwards.", order = 3)
+    private @Nullable File scenario;
```

Comment:
If this is required you can set the required option to enforce that the option is used.

```suggestion
    @CommandLine.Option(names = {"-f", "--file"}, required = true, description = "The scenario file. " +
            "If a scenario file is given this command uploads, executes and deletes the scenario afterwards.", order = 3)
    private @Nullable File scenario;
```

Reply:
I think, this would make this '@NotNull' and make the not null check later on redundant.